### DISCUSSION TITLE
How can i custom tesseract parameters?

### DISCUSSION THREAD

**Discussion by hoolyfook** [ocrmypdf/OCRmyPDF#1417] [Q&A] [answered]
default params not working on my pdf, so i need to custom it
`tesseract page_1.png page_1 -l script/Vietnamese+vie --oem 3 --psm 6 -c preserve_interword_spaces=1`

**Comment by jbarlow83** [comment, marked as answer]
You can create a text file e.g. named `my.cfg` containing

```
preserve_interword_spaces=1
```

then use

`ocrmypdf --tesseract-config my.cfg --tesseract-oem 3 --tessearct-psm 6 --lang script/Vietnamese --lang vie in.pdf out.pdf`
